daprdocs/content/en/reference/components-reference/supported-workflow-backend/sqlite-workflow-backend.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
Personally, I don't think we should document anything that isn't supported yet. Could be confusing to users. @msfussell wdyt?

Reply:
Yes, I agree. If this has not been implemented it will confuse people 
@ASHIQUEMD - Can we take this out and put this into a future docs issue, if and when this get implemented?